### PR TITLE
Feature detection

### PR DESCRIPTION
**Pull request by tgerke**
Currently broken, PR for help/previews in netlify

### CODE REVIEW

**Comment by gadenbuie** [comment]
Okay, as I understand it (I might be wrong), modernizr will add the `no-offset-path` class on a top-level element, like the `<html>` element. So rather than placing the `no-offset-path` CSS class on the element that needs the alternate rules, you want to write a CSS rule for a that element as a child of the `no-offset-path` classed-element.

You can target something like this:

```
<html class="no-offset-path">
  ...
     ...
        <div class="rocket"></div>

</html>
```

with a CSS rule 

```css
.no-offset-path .rocket {
  /* rules for the .rocket 
     that only apply when modernizr 
     adds the .no-offset-path higher up */
}
```

So basically, you can try removing the `@media` query around this rule https://github.com/tgerke/build-your-own-universe/blob/a0c9489188d63ce1dd1f6fd65657e9ec14134ea3/docs/css/build-your-universe.css#L71-L77
and change the CSS rule inside to `.no-offset-path .rocket`.

It won't make the rocket animate, but it will make sure the rocket shows up when the browser doesn't support the `offset-path` property.

**Comment by gadenbuie** [comment]
Btw, those CSS lines are what's called a _browserhack_. Ideally that CSS will only work on Safari browsers:

```css
 @media not all and (min-resolution:.001dpcm) { @media { 
  /* rules that only apply in Safari browsers */
}}
```

**Comment by gadenbuie** [comment]
One other challenge you'll likely run into, I've minimized the `build-your-universe.css` CSS. For dev and preview, you'll need to change `build-your-universe.min.css` in the slide's YAML to `build-your-universe.css`.

If you want to re-minimize the CSS, here's how I did it using [clean-css-cli](https://www.npmjs.com/package/clean-css-cli)

```bash
# install the cleancss-cli node package
npm install clean-css-cli -g

cd docs/css

# minimize the css
cleancss --compatibility '*' --source-map -o build-your-universe.min.css build-your-universe.css
```
